pom.xml

```diff
@@ -55,11 +55,19 @@
     </dependency>
     <dependency>
       <groupId>io.quarkus</groupId>
-      <artifactId>quarkus-resteasy-jackson</artifactId>
+      <artifactId>quarkus-resteasy-reactive</artifactId>
```

Comment:
This is resolved transitively through `quarkus-resteasy-reactive-jackson`

Reply:
Cool, I'll remove